models/taskRequests.js

```diff
@@ -338,10 +338,74 @@ const approveTaskRequest = async (taskRequestId, user) => {
   }
 };
 
+const addNewFields = async () => {
+  const taskRequestsSnapshots = (await taskRequestsCollection.get()).docs;
+
+  const bulkWriter = firestore.bulkWriter();
+  let documentsModified = 0;
+  const totalDocuments = taskRequestsSnapshots.length;
+
+  await Promise.all(
+    taskRequestsSnapshots.map(async (taskRequestsSnapshot) => {
+      const taskRequestData = taskRequestsSnapshot.data();
+      if (!taskRequestData.requestType) {
+        const { taskData } = await tasksModel.fetchTask(taskRequestData.taskId);
+        const usersRequestList = taskRequestData.requestors.map((requestorId) => {
+          let userStatus = TASK_REQUEST_STATUS.PENDING;
+
+          if (taskRequestData.status === TASK_REQUEST_STATUS.APPROVED && requestorId === taskRequestData.approvedTo) {
+            userStatus = TASK_REQUEST_STATUS.APPROVED;
+          }
+
+          return {
+            userId: requestorId,
+            status: userStatus,
+          };
+        });
+        const updatedTaskRequestData = {
+          ...taskRequestData,
+          requestType: TASK_REQUEST_TYPE.ASSIGNMENT,
```

Comment:
can the old taskreq not be task creation as we are assigning assignment by default.